bigtable/tests/mutation_integration_test.cc

```diff
@@ -0,0 +1,508 @@
+   */
+  void DeleteFromColumn(bigtable::Table& table, bigtable::Cell cell) {
+    bigtable::SingleRowMutation mutation(cell.row_key());
+    mutation.emplace_back(bigtable::DeleteFromColumn(cell.family_name(),
+                                                     cell.column_qualifier()));
+
+    bigtable::BulkMutation bulk;
+    bulk.emplace_back(std::move(mutation));
+    table.BulkApply(std::move(bulk));
+  }
+
+  /*
+   * Delete the records from table for the specified family and column with
+   * timestamp starting from time_begin.
+   * time_begin: This parameter is inclusive, i.e. timestamp >= time_begin
+   */
+  void DeleteFromColumnStartingFrom(bigtable::Table& table, bigtable::Cell cell,
+                                    std::int64_t time_begin) {
+    bigtable::SingleRowMutation mutation(cell.row_key());
+    mutation.emplace_back(bigtable::DeleteFromColumnStartingFrom(
+        cell.family_name(), cell.column_qualifier(), time_begin));
+
+    bigtable::BulkMutation bulk;
+    bulk.emplace_back(std::move(mutation));
+    table.BulkApply(std::move(bulk));
+  }
+  /*
+   * Delete the records from table for the specified family and column with
+   * timestamp ending at time_end.
+   * time_end  : This parameter is not inclusive. i.e. timestamp < time_end
+   */
+  void DeleteFromColumnEndingAt(bigtable::Table& table, bigtable::Cell cell,
+                                std::int64_t time_end) {
+    bigtable::SingleRowMutation mutation(cell.row_key());
+    mutation.emplace_back(bigtable::DeleteFromColumnEndingAt(
+        cell.family_name(), cell.column_qualifier(), time_end));
+
+    bigtable::BulkMutation bulk;
+    bulk.emplace_back(std::move(mutation));
+    table.BulkApply(std::move(bulk));
+  }
+  /*
+   * Delete the records from table for the specified family and column with
+   * timestamp starting from time_begin.
+   * time_begin: This parameter is inclusive i.e. timestamp >= time_begin
+   * time_end  : This parameter is not inclusive. i.e. timestamp < time_end
+   */
+  void DeleteFromColumn(bigtable::Table& table, bigtable::Cell cell,
+                        std::int64_t time_begin, std::int64_t time_end) {
+    bigtable::SingleRowMutation mutation(cell.row_key());
+    mutation.emplace_back(bigtable::DeleteFromColumn(
+        cell.family_name(), cell.column_qualifier(), time_begin, time_end));
+
+    bigtable::BulkMutation bulk;
+    bulk.emplace_back(std::move(mutation));
+    table.BulkApply(std::move(bulk));
+  }
+
+  /*
+   * Delete records from a specific column_family
+   */
+  void DeleteFromFamily(bigtable::Table& table, std::string row_key,
+                        std::string column_family) {
+    bigtable::SingleRowMutation mutation(row_key);
+    mutation.emplace_back(bigtable::DeleteFromFamily(column_family));
+
+    bigtable::BulkMutation bulk;
+    bulk.emplace_back(std::move(mutation));
+    table.BulkApply(std::move(bulk));
+  }
+
+  /*
```

Comment:
If this is supposed to be a Doxygen comment it is missing the `/**` at the beginning.